deploy/cloudformation/README.md

```diff
@@ -5,18 +5,27 @@ This CloudFormation template creates a role for elastic-agent and attaches it to
 The EC2 instance has elastic-agent preinstalled in it using the fleet URL and enrollment token.
 
 ### How to test it
-The template can be tested with AWS SDK as follows:
-Create a `.env` file of the form:
+*Prerequisites:*
+1. You have an elastic stack deployed in the cloud that includes Kibana, elasticsearch and fleet-server (check https://cloud.elastic.co/ to deploy your own stack)
+2. You have AWS CLI installed on your laptop and configured to work with our dev account `elastic-security-cloud-security-dev` (in particular, `~/.aws/config` and `~/.aws/credentials` should be set, check https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html for more information)
+
+*Steps:*
+1. Install Vulnerability Management integration on a new agent policy, you might have to check the "Display beta integrations" checkbox.
+2. After you installed the integration you can install a new elastic-agent, you should keep the fleet URL and the enrollment token.
+3. On cloudbeat repo, create a `deploy/cloudformation/.env` file of the form:
```

Comment:
nit can you please add an example stack name? so others would follow

like
`STACK_NAME="your unique stack name" # john-qa-bc2-8-9-0-May28`